<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAnimal</class>
 <widget class="QDialog" name="EditAnimal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>559</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>16</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>16</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>16</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>16</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>145</red>
        <green>145</green>
        <blue>145</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="cleanlinessSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cleanlinessLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="aggressionLabel_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Aggression Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="typeLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="attachmentSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="attachmentLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="attachmentLabel_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Attachment Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="aggressionLabel_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Comfort with Children (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="labelColour_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Colour</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="loudnessSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loudnessLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="typeComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dog</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bird</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hamster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mouse</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Guinea Pig</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelBreed_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Breed</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="labelColour_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cleanliness Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="obedienceSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="obedienceLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="childrenComfortSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="childrenComfortLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="aggressionSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aggressionLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="attachmentLabel_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Loudness Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelBreed_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Obedience Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="breedComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select a Type</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="ageLabel_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Intelligence Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QComboBox" name="colourComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>White</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grey</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="13" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="10,2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QSlider" name="intelligenceSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intelligenceLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="ageLabel_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QComboBox" name="sizeComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="sizeLabel_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="ageLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="sexLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Sex</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="sexComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="energyLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Energy Level</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="energyComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labelBreed_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Flat Cost</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="costLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="sexLabel_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cost per Year</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="costPerYearLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="labelColour_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Hypoallergenic</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QRadioButton" name="isCrateTrainedRadioButtonYES">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isCrateTrainedRadioButtonNO">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_3</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="ageLabel_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Crate Trained</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QRadioButton" name="isHypoallergenicRadioButtonYES">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isHypoallergenicRadioButtonNO">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="sizeLabel_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Neutered/Spayed</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QRadioButton" name="isNeuteredRadioButtonYES">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isNeuteredRadioButtonNO">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>669</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Animals are Currently Available</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditAnimal</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditAnimal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
